Jenkinsfile.hardware

```diff
@@ -0,0 +1,133 @@
+#!groovy
```

Comment:
- I would suggest to change the name from Jenkinsfile.hardware to Jenkinsfile.SensorboxTests (as it seems that this is the commonly used name for that kind of tests), so it's more likely that people know what to expect inside of that file
- Please remove all those empty lines, so it is closer and thus easier to compare to other Jenkinsfiles
- General comment: not meant for a change, just for the sake of addressing it. I'm fine with changing from tabs (currently the other Jenkinsfiles) to spaces.